.github/workflows/docker.yml

```diff
@@ -8,18 +8,38 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
-      - name: Publish to Github Docker Package Registry
-        uses: elgohr/Publish-Docker-Github-Action@2.21
+      - name: Get the version
+        id: get_version
+        run: |
+          VERSION=${GITHUB_REF#refs/tags/}
```

Comment:
consider to add `tags: v*` in this workflow, and remove docker pushing in `release.yaml`

Reply:
sure, you could do that in another pr